src/renderers/sigma.renderers.canvas.js

```diff
@@ -433,9 +433,9 @@
   sigma.renderers.canvas.prototype.clear = function() {
     var k;
 
-    for (k in this.domElements)
-      if (this.domElements[k].tagName === 'CANVAS')
-        this.domElements[k].width = this.domElements[k].width;
+    for (k in this.contexts) {
```

Comment:
because we have to use the context instead of the domElement to use the drawing functions